pycontrails/models/accf.py

```diff
@@ -377,3 +375,30 @@ def _update_accf_config(self) -> None:
             "save_path": "./",
             "save_format": "netCDF",
         }
+
+
+def _rad_instantaneous_to_accumulated(ds: xr.Dataset) -> xr.Dataset:
+    """Convert instantaneous radiation to accumulated radiation."""
+
+    for name, da in ds.items():
+        try:
+            unit = da.attrs["units"]
+        except KeyError as e:
+            msg = (
+                f"Radiation data contains '{name}' variable "
+                "but units are not specified. Provide units in the "
+                f"rad['{name}'].attrs passed into ACCF."
+            )
+            raise KeyError(msg) from e
+
+        if unit == "J m**-2":
+            continue
+        if unit != "W m**-2":
+            msg = f"Unexpected units '{unit}' for '{name}'. Expected 'J m**-2' or 'W m**-2'."
+            raise ValueError(msg)
+
+        # Convert from W m**-2 to J m**-2
+        da.attrs["units"] = "J m**-2"
```

Comment:
I think these two lines are here in error

Reply:
I think this is what I want ... but I might have my eyes crossed. I slightly changed this to 9878bec874ca2c8ee0e5175c856a121ad17b0d5e but that is only a syntax change.

If we end up on this line of code, that means the units are `W m**-2`. So I'm reassigning them to `J m**-2`, then multiplying by 3600 to make the conversion on the data itself.

Reply:
This was my mis-reading. Your new implementation is much clearer.